players/pocker_ai.py

```diff
@@ -5,6 +5,9 @@
 
 
 class PokerAi(ABC):
+    def __init__(self, player_number: bool):
+        self.player_number: bool = player_number
```

Comment:
why player number is not an int?
if it is a bool, maybe `is_first_player`

Reply:
I think a bool will do